modules/mobile-agents/android/jax-rs/src/main/java/org/wso2/carbon/mdm/services/android/OperationMgtService.java

```diff
@@ -197,8 +205,8 @@ public Response removePassword(@HeaderParam("Accept") String acceptHeader,
     @POST
     @Produces(MediaType.APPLICATION_JSON)
```

Comment:
While we allow any type of response via the accept header, why do we also limit response to JSON?